test/utils.ts

```diff
@@ -146,4 +146,4 @@ export function ignoreWebSocketDisconnect(error: Error): void {
     return
   }
   throw error
-}
```

Comment:
To avoid confusion, I suggest renaming this `testUtils`, so it doesn't seem like this is testing the utils folder.